### DISCUSSION TITLE
Can't resolve 'bufferutil' and 'utf-8-validate'

### DISCUSSION THREAD

**Discussion by wss29** [eclipse-glsp/glsp#1079] [Q&A] [answered]
Hi,
    I download glsp-server-node-1.1.0-RC10 source code from https://github.com/eclipse-glsp/glsp-server-node/releases, I want to build and try it. 
   When I yarn I get follow note
![image](https://github.com/eclipse-glsp/glsp/assets/4537809/e81972cf-4351-4192-9b56-3292e83de706)
I google the problem about 'bufferutil ' and 'utf-8-validate', and someone says that use follow 
![image](https://github.com/eclipse-glsp/glsp/assets/4537809/fa6fec24-cbec-41b7-a9da-d01f32a9267c)
but I try it unuseful. Although it can generate wf-glsp-server-node.js the error look not good. Does anyone know how to make it disappear?

**Comment by tortmayr** [comment, marked as answer]
Hi @wss29,
technically this is not an error and  "just" a warning. As you already mentioned it does not affect the build result and the generated node server works as expected. Nevertheless, I agree that we should try to get rid of the warning by fixing the root cause.
In the meantime, you could also just try to filter out the warning by adding a matching regex to the `ingoreWarnings` array in the webpack config.
